### PR TITLE
Feature/lgcs 322_2 removing changes that caused develop to error

### PR DESCRIPTION
**Pull request by h0usemaintenance**

### CODE REVIEW

File: scripts/setup-postgres.sh [hunk 1]
```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-# Create the database users expected along with databases
-sudo -u postgres psql postgres -c "CREATE USER $DATABASE_USER WITH PASSWORD '$DATABASE_PASSWORD' CREATEDB;"
```
Comment (joel-stobart) [inline scripts/setup-postgres.sh:4]: Are we sure we want to delete this?

File: scripts/travis/before_script.sh [hunk 2]
```diff
@@ -1,10 +1,6 @@
 #!/bin/bash
 
 APPLICATIONS=("CyberHealth")
-
-# shellcheck disable=SC1091
- source scripts/setup-postgres.sh
```
Comment (joel-stobart) [inline scripts/travis/before_script.sh:6]: Are you sure?

**Review by joel-stobart — APPROVED** [review]

File: scripts/setup-postgres.sh (hunk 1 above)
Comment (h0usemaintenance) [reply scripts/setup-postgres.sh:4]: For now only

File: scripts/travis/before_script.sh (hunk 2 above)
Comment (h0usemaintenance) [reply scripts/travis/before_script.sh:6]: For now only.

**Pull request merged**